arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -300,7 +299,8 @@ class ffmpeg_video_input::priv
     }
     this->frame_advanced = 0;
     this->f_frame->data[0] = NULL;
-
+    avcodec_close(codec_context_origin);
```

Comment:
I'm not as convinced that this need. `codec_context_origin` was not allocated by us.  It is a pointer to codec memory that was already allocated elsewhere.  I would think this would be closed/deleted automatically when we clean up `this->f_format_context`.  Can you confirm that this line makes a difference in valgrind?

Reply:
Just checked it does not. I think my main motivation there was to find all codec contexts and close them explicitly. Will remove the close